third_party/openocd/repos.bzl

```diff
@@ -0,0 +1,23 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//rules:repo.bzl", "http_archive_or_local")
+
+def openocd_repos(local = None):
+    http_archive_or_local(
+        name = "openocd",
+        local = local,
+        url = "https://sourceforge.net/projects/openocd/files/openocd/0.12.0-rc1/openocd-0.12.0-rc1.tar.gz",
+        strip_prefix = "openocd-0.12.0-rc1",
+        build_file_content = """
```

Comment:
Nit: I preferred the prior setup here: `build_file = "//third_party/openocd/BUILD.openocd.bazel"`.

If you want to keep this, please make this a constant rather than here-doc style:
```
_BUILD_FILE_CONTENT = """
...
"""
```

Reply:
Ah, I don't feel strongly about it. Done.